charts/kargo/templates/api/deployment.yaml

```diff
@@ -45,16 +45,6 @@ spec:
               value: {{ .Values.api.protocol }}://{{ .Values.api.host }}
             - name: ADMIN_ACCOUNT_TOKEN_AUDIENCE
               value: {{ .Values.api.host }}
-            - name: ADMIN_ACCOUNT_PASSWORD_HASH
-              valueFrom:
-                secretKeyRef:
-                  name: kargo-api
-                  key: adminPassword
-            - name: ADMIN_ACCOUNT_TOKEN_SIGNING_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: kargo-api
-                  key: tokenSigningKey
             - name: ADMIN_ACCOUNT_TOKEN_TTL
               value: {{ .Values.api.adminAccount.tokenTTL }}
```

Comment:
So I know that I only mentioned Secret in the original issue, but I realize now I also meant to extend this to our non-sentive settings. WDYT about doing the same thing with envFrom but having our config centralized in a `kargo-api` ConfigMap?

Reply:
Ah... I didn't misunderstand it as only applying to secrets... but I definitely did misunderstand the idea that _all_ the logic for what env vars are defined / not defined / and how they're defined would all move to the secrets/configmaps.

That makes a lot of sense.

Will fix.

Reply:
Ok... I've made the change more encompassing. We don't use `env:` anywhere now. We only use `envFrom:` and reference applicable secret and/or configmap.